src/api/client.ts

```diff
@@ -0,0 +1,30 @@
+import AsyncStorage from "@react-native-async-storage/async-storage"
+import { cacheExchange, createClient, dedupExchange, errorExchange, fetchExchange } from "urql"
+import { authExchange } from "@urql/exchange-auth"
+import { AuthState, addAuthToOperation, getAuth, willAuthError, didAuthError } from "./auth"
+import { NavigationHelpersContext } from "@react-navigation/native"
+
+export function createAPIClient() {
+    return createClient({
+        url: "https://api.loop.page",
+        exchanges: [
+            dedupExchange,
+            cacheExchange,
+            errorExchange({
+                onError: async (error) => {
+                    const isAuthError = error.graphQLErrors.some((e) => e.message.includes("[uar]"))
+                    if (isAuthError) {
+                        // await AsyncStorage.removeItem("token")
```

Comment:
Is this commented out for a reason?

Reply:
It was commented out during the multi-user session debugging. I'll put it back.